server/tests/factory/article-factory.js

```diff
@@ -10,7 +10,6 @@ const createTestArticle = async (userId, { title, body }) => {
     slug: faker.lorem.slug(),
     title: title || faker.lorem.words(),
     body: body || faker.lorem.sentences(),
-    image: faker.image.imageUrl(),
```

Comment:
any reason why this is removed though?

Reply:
I think this happened when I was resolving a Merge conflict. I will put that back ASAP